addons/website_portal/controllers/main.py

```diff
@@ -55,7 +55,8 @@ def _get_archive_groups(self, model, domain=None, fields=None, groupby="create_d
     @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
     def account(self, **kw):
         values = self._prepare_portal_layout_values()
-        return request.render("website_portal.portal_my_home", values)
+        response = request.render("website_portal.portal_my_home", values)
```

Comment:
Reads like an unnecessary change to me ;) . I suppose some leftover of code debugging.